content/docs/projects/tdd.md

```diff
@@ -0,0 +1,602 @@
+---
+title: "Test-Driven Development Guidelines"
+description: "Know more about our guidelines when writing and running unit tests."
+lead: "Author/s: Gab Barbudo, Celyn Raquel"
+date: 2023-07-05T14:30:18+08:00
+lastmod: 2023-07-05T14:30:18+08:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "projects"
```

Comment:
Hmm, would it be better to put this under the same folder/section as the other standards?